src/ClaimService/Controllers/CategoriesController.cs

```diff
@@ -27,33 +34,50 @@ public CategoriesController(IMediator mediator)
   /// <summary>
   /// Create category.
   /// </summary>
+  /// <param name="command">Category to create.</param>
   [HttpPost]
+  [SwaggerOperationFilter(typeof(TokenOperationFilter))]
+  [ProducesResponseType(typeof(Guid), StatusCodes.Status201Created)]
+  [ProducesResponseType(typeof(ErrorResponse), StatusCodes.Status400BadRequest)]
+  [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+  [ProducesResponseType(typeof(ErrorResponse), StatusCodes.Status403Forbidden)]
```

Comment:
also add 404 and add for put request too

Reply:
404 for create?

Reply:
I'am sorry, i meant edit request